contrib/examples/walkthrough/ups-instance-default-sp.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
```

Comment:
I agree that it would be a great story, I'm just worried when a user tries to create their own instance they won't have a complete example. Any case, I'd prefer to keep this here, but again if you feel strongly, I'll yank it.